eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -132,7 +132,7 @@ jobs:
         inputs:
           mavenPomFile: pom.client.xml
           options: '$(DefaultOptions) -Djava-lts -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true'
-          mavenOptions: '$(LoggingOptions)'
+          mavenOptions: '-Xmx3072m $(LoggingOptions)'
```

Comment:
Any particular reason you picked this value? What is the default?

Reply:
Its in use elsewhere in the build. I can't remember where I got this value from - but I think I saw it elsewhere in our builds somewhere. Could potentially reduce or increase but this seems to work.

I did a little bit of reading. Some sites suggest that the allocated memory is 1/4 of system ram. The hosted VMs have ~7GB of RAM. So this would lift that quite a bit.

Reply:
Any other issues @weshaggard  would be good to unblock @g2vinay

Reply:
No issues. I was just curious about why this particular number was chosen.